Makefile

```diff
@@ -5,7 +5,7 @@ CURRENT_VERSION = $(shell git log --merges --oneline | perl -ne 'if(m/^.+Merge p
 
 BUILD_FLAGS = -ldflags "\
 	      -s -w \
-	      -X main.Version \"$(VERSION)\" \
+	      -X \"main.Version=$(VERSION)\" \
```

Comment:
[note] it's not related with this p-r, but it seems we don't use `main.Version`.
It was introduced by this p-r https://github.com/mackerelio/mackerel-agent-plugins/pull/6, but it might be just included by mistake...?

I guess we can remove this in this p-r or following p-r.